package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "coding-standard",
+  "description": "ZooRoyal Coding Standard package.json",
+  "private": true,
+  "homepage": "https://github.com/ZooRoyal/coding-standard",
+  "engines": {
+    "node": "^6.9.5"
+  },
+  "dependencies": {
```

Comment:
ich habe in DEV-2037 eine verschlankte package.json eingefügt. manche libs brauchen wir hier nicht.

Reply:
ok. ich rebase. hast du das schon gepushed?